langstream-runtime/langstream-runtime-impl/src/main/java/ai/langstream/runtime/agent/AgentRunner.java

```diff
@@ -140,28 +146,29 @@ public static void run(RuntimePodConfiguration configuration,
 
         log.info("Starting agent {} with configuration {}", agentId, configuration.agent());
 
-        TopicConnectionsRuntime topicConnectionsRuntime =
-                TOPIC_CONNECTIONS_REGISTRY.getTopicConnectionsRuntime(configuration.streamingCluster());
+        try (NarFileHandler narFileHandler = new NarFileHandler(agentsDirectory);) {
```

Comment:
So now we only support loading NARs ? Or is it possible to also load JARs being aware that we lose isolation ?

Reply:
this is for internal agents only at the moment, at some point we could add custom Java agents
you can already bundle custom jars (for Kafka Connect and for JDBC)